tst/testinstall/ConjNatSym.tst

```diff
@@ -6,7 +6,7 @@ gap> IsConjugate( SymmetricGroup(5), Group((1,2)), Group((3,4,5)));
 false
 
 # This runs into the TryNextMethod case
-gap> IsConjugate( SymmetricGroup(200),PrimitiveGroup(200,4), PrimitiveGroup(200,3));
+gap> IsConjugate( SymmetricGroup(200),SymmetricGroup(200), AlternatingGroup(200));
```

Comment:
I think this defeats the point of this test, which was to use copies of the symmetric and alternating group for which the filters `IsNaturalAlternatingGroup` resp. `IsNaturalSymmetricGroup` are not set.

Most likely this could also be achieved by doing something like `Group(GeneratorsOfGroup(SymmetricGroup(200)))` to "forget" everything about that group.

Reply:
I initially thought that that would probably be the case too, @fingolfin. But actually `PrimitiveGroup(200,4)` and `PrimitiveGroup(200,3)` already know exactly what groups they are:
```gap
gap> G := PrimitiveGroup(200, 4);;
gap> HasIsNaturalSymmetricGroup(G) and IsNaturalSymmetricGroup(G);
true
gap> H := PrimitiveGroup(200, 3);;
gap> HasIsNaturalAlternatingGroup(H) and IsNaturalAlternatingGroup(H);
true
```
So we don't lose anything by moving to `SymmetricGroup(200)` and `AlternatingGroup(200)`.